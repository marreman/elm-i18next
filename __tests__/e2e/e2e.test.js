const main = require("../../src/main")
const translations = require("./translations.json")

test("main", () => {
  const actual = main(translations, "Translations", __dirname)

})
