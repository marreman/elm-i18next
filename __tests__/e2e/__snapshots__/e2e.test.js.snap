// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main 1`] = `
"module TestTranslations.Account exposing (..)


email =
    \\"Email\\"


error =
    \\"Something went wrong, try reloading the page.\\"


text2hOfferMessage =
    \\"Something that begins with a number\\"
"
`;

exports[`main 2`] = `
"module TestTranslations.Signup exposing (..)


incognitoMessage =
    \\"This is a message\\"


loading =
    \\"Loading\\"
"
`;

exports[`main 3`] = `
"module TestTranslations.Signup.NameStep exposing (..)


label =
    \\"First name\\"


title =
    \\"Enter your name\\"
"
`;
